app/components/EventInfo.tsx

```diff
@@ -0,0 +1,98 @@
+import { View, ViewStyle } from "react-native"
+import React from "react"
+import { Icon } from "./Icon"
+import { colors } from "../theme"
+import { Text } from "./Text"
+import { Event } from "../../types/trip"
+import { Avatar, Modal } from "@ui-kitten/components"
+import { Button } from "./Button"
+
+export function getTime(date: Date) {
+  // Need to check for minute rather than just put hour
+  return date.toLocaleTimeString([], { hour: "2-digit" })
+}
+
+type EventInfoProps = {
+  event: Event
+}
+export function EventInfo(props: EventInfoProps) {
+  const { event } = props
+  const [visible, setVisible] = React.useState(true)
+
+  return (
+    <Modal
+      visible={visible}
+      style={{ width: "80%", height: "80%", justifyContent: "center", borderRadius: 5 }}
+    >
+      <View style={{ backgroundColor: "white", borderRadius: 20, padding: 15 }}>
+        <View>
+          <Text text={event.title} preset="title" style={{ color: colors.palette.neutral1000 }} />
+          <View style={{ flexDirection: "row", justifyContent: "space-between" }}>
+            <View style={{ flexDirection: "row" }}>
+              <Icon icon="pin" />
+              <Text
+                text={event.location}
+                preset="bold"
+                size="sm"
+                style={{ color: colors.palette.neutral1000, paddingLeft: 5 }}
+              />
+            </View>
+            <Text
+              text={`${getTime(event.duration.start)} - ${getTime(event.duration.end)}`}
+              preset="bold"
+              size="sm"
+              style={{
+                textAlign: "right",
+                color: colors.palette.neutral1000,
+              }}
+            />
+          </View>
+
+          {/* <Icon icon="personSmall" style={{left:300}}/> */}
+          {/* This will be the icons spot */}
+        </View>
+        <View style={{ flexDirection: "row", justifyContent: "space-evenly", padding: 10 }}>
+          <Avatar
```

Comment:
Return these in a map so it'll be easier to integrate later when we have the backend connected.

I think the attendee for an event is an array. So for now, I'd suggest doing something like 
```
{
{event. attendees.map((attendees) => {
< Avatar source={{uri.......}} />
})
```
You can keep the URI a constant though, cause wed probably have something like trip.attendees.get(attendee).profilePic later to get the images